lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_emulator_helper.rb

```diff
@@ -0,0 +1,198 @@
+module Fastlane
+  module Helper
+    module Android
+      # Helper methods to manipulate System Images, AVDs and Android Emulators
+      #
+      class EmulatorHelper
+        BOOT_WAIT = 2
+        BOOT_TIMEOUT = 60
+
+        SHUTDOWN_WAIT = 2
+        SHUTDOWN_TIMEOUT = 60
+
+        def initialize
+          @tools = Fastlane::Helper::Android::ToolsPathHelper.new
+        end
+
+        # Installs the system-image suitable for a given Android `api`, with `google_apis`, and for the current machine's architecture
```

Comment:
Nitpick: To make this consistent with the usage [below](https://github.com/wordpress-mobile/release-toolkit/pull/409/files#diff-d271fcb6de300236feaec1c6b889596caf09c69201f3f8d89a692cd0496dd8ddR170)


```suggestion
        # Installs the system-image suitable for a given Android `api`, with Google APIs, and for the current machine's architecture
```

Personally, I prefer "Google APIs" over `google_apis` because the latter makes me think of a method parameter.

Reply:
Right. I used this wording for `google_apis` here because that's how it's specified in the system-images qualifier expected to be passed to `sdkmanager --install <qualifier>`. But I can see the rationale for consistency with below and with differentiating `Google APIs` as a name from `api` as a parameter (and also the image qualifier could be seen as an implementation details… even if I thought the explicitness of the package component specifier here could help explain better which package this method installed)

```
sdkmanager --list_installed
…
system-images;android-28;google_apis;arm64-v8a | …
…